cpt/packager.py

```diff
@@ -249,6 +250,12 @@ def __init__(self, username=None, channel=None, runner=None,
         else:
             self.docker_32_images = os.getenv("CONAN_DOCKER_32_IMAGES", False)
 
+        docker_args = docker_args or split_colon_env("CPT_DOCKER_ARGS")
+
+        if docker_args is None:
+            docker_args = []
+        self.docker_args = docker_args
```

Comment:
If it shouldn't be false it is better to do `self.docker_args = docker_args or []`